### PR TITLE
Updated to .Net 6.0 and Ahead of time compilation for Linux-x64

### PR DESCRIPTION
**Pull request by sharpninja**
* Updated to .Net 5.0
* Added Publish Profile for Linux-x64
* Added Publish.ps1 to easily build published output.

### CODE REVIEW

File: dotnet-lambda/global.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
   "sdk": {
-    "version": "3.1.412"
+    "version": "5.0.x"
```
Comment (sharpninja) [inline dotnet-lambda/global.json:3]: Always uses the latest .Net 5.

**Comment by Aleksandr-Filichkin** [comment]
Hi @sharpninja 
Thank you for MR, but it cannot be merged. Because MR should have a fully working solution.
If you want to help with .Net5.0 then please create a fully working code and do not update 3.1 version(it's better to create a separate lambda)
Thank you !

**Comment by sharpninja** [comment]
Are there scripts to build out the complete environment?

**Comment by sharpninja** [comment]
> Hi @sharpninja Thank you for MR, but it cannot be merged. Because MR should have a fully working solution. If you want to help with .Net5.0 then please create a fully working code and do not update 3.1 version(it's better to create a separate lambda) Thank you !

Looking at all of the files in the repository, it looks like you really need a Docker-based solution for .Net so that's what I'll focus on.

**Comment by odinnou** [comment]
I think you could go to .NET 6

**Comment by sharpninja** [comment]
* .Net 6.0!
* Docker!
* Deploys Correctly!

**Comment by sharpninja** [comment]
Wait, code needs to have logging removed.

**Comment by sharpninja** [comment]
Removed the logging.

**Pull request closed without merging**